app/migrations/Version20190116162854.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+final class Version20190116162854 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE donations ADD nationality VARCHAR(2) DEFAULT \'FR\' NOT NULL');
+        $this->addSql('ALTER TABLE donations CHANGE nationality nationality VARCHAR(2) NOT NULL');
```

Comment:
I don't get the second line, was it generated by a diff ?

Reply:
the first one adds a new column with a default value for existing rows and the second one remove this default value